internal/util/validation/validation.go

```diff
@@ -70,6 +71,28 @@ func ValidateString(s *string, path string, minLen int, maxLen int, patternRegex
 	return asErrors(errs)
 }
 
+func ValidateBase64Field(field *string, maxLenMB int) []error {
```

Comment:
The other validations all measure length in bytes, so having MB here could easily lead to confusion. I'd suggest using bytes here and call the function with `...(config.CaCrt, 20*1024*1024)` above.

Reply:
I was calculating an expected length for a base64 encoded CA or cert/key. It's true that the rest are in bytes, but I thought having a long number there would be too ugly. I could get along with * :-)

Reply:
Readability > beauty. You could, of course, define a `const maxBase64CertificateLength = 20*1024*1024` or so, then you'd have both beauty and readability :)